client/cmd/bisonw-desktop/winres.json

```diff
@@ -17,22 +17,22 @@
     "#1": {
       "0000": {
         "fixed": {
-          "file_version": "1.0.4.0",
-          "product_version": "1.0.4.0 pre"
+          "file_version": "1.0.0.0",
+          "product_version": "1.1.0.0 pre"
         },
         "info": {
           "0409": {
             "Comments": "",
             "CompanyName": "Decred",
             "FileDescription": "Bison Wallet (WebView version)",
-            "FileVersion": "1.0.4",
+            "FileVersion": "1.0.0.0",
```

Comment:
I don't think we need to change this past 1.0.0.0 which it is currently in v1.0.5. https://github.com/decred/dcrdex/blob/37585833528544f80dddd92dcdff10a78ad01e1f/client/cmd/bisonw-desktop/winres.json#L20

Reply:
It looks like that are
```
Major Version
Minor Version
Build Number
Revision Number
```
So major and minor line up. I think its fine to have the third one patch and ignore the fourth.

But, researching some more, I guess we should just update the file version to current version. It doesn't seem to do much but I do see that version in the installed apps.